Explorer/Assets/DCL/PerformanceAndDiagnostics/DebugUtilities/Builders/IDebugContainerBuilder.cs

```diff
@@ -13,10 +14,28 @@ public interface IDebugContainerBuilder
 
         DebugContainer Container { get; }
 
-        DebugWidgetBuilder AddWidget(string name);
+        Result<DebugWidgetBuilder> AddWidget(string name);
 
         IReadOnlyDictionary<string, DebugWidget> Widgets { get; }
 
         void BuildWithFlex(UIDocument debugRootCanvas);
+
+        public static class Categories
+        {
+            public const string ROOM_INFO = "Room: Info";
+            public const string ROOM_ISLAND = "Room: Island";
+            public const string ROOM_SCENE = "Room: Scene";
+            public const string PERFORMANCE = "Performance";
+            public const string MEMORY = "Memory";
+        }
+    }
+
+    public static class DebugContainerBuilderExtensions
+    {
+        public static DebugWidgetBuilder? TryAddWidget(this IDebugContainerBuilder debugContainerBuilder, string name)
```

Comment:
Do you need this extra logic? 
Since you are not using the error message when widget is not successful, maybe you directly use `IDebugContainerBuilder.AddWidget` and make it return a `DebugWidgetBuilder?`

Reply:
It's good to have it, since it gives more descriptive contracts, TryAddWidget is just a wrap to prevent boilerplate in places where it should be optional